### PR TITLE
solution

### PR DESCRIPTION
**Pull request by 1stIndominus**
[DEMO LINK](https://1stIndominus.github.io/js_promise_basic_DOM/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+const message = (className, text) => {
+  const div = document.createElement('div');
+
+  div.className = className;
+  div.textContent = text;
+  document.body.append(div);
+};
+
+function promise1() {
+  return new Promise((resolve) => {
```
Comment (polosanya) [inline src/scripts/main.js:14]: Do you need function to create promise?
Just save the promise to some variable

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const message = (className, text) => {
```
Comment (polosanya) [inline src/scripts/main.js:5]:
```suggestion
const showMessage = (className, text) => {
```
image

**Review by polosanya — CHANGES_REQUESTED** [review]
Let's try to pass the tests

File: src/scripts/main.js [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+  }, 3000);
+});
+
+promise1
+  .then(result => showMessage('message', result))
+  .catch(error => showMessage('error-message', error));
+
+promise2
+  .then(result => showMessage('message', result))
+  .catch(error => showMessage('error-message', error));
```
Comment (Anastasiia-Svintsova) [inline src/scripts/main.js:31]: You do the same for both promises here. Create a function for it, please

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
Good! Fix one small comment, please

File: src/scripts/main.js [hunk 4]
```diff
@@ -1,3 +1,32 @@
...
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+const promiseResult = (promise) => {
```
Comment (polosanya) [inline src/scripts/main.js:25]:
```suggestion
const handlePromises = (promise) => {
```

**Review by polosanya — APPROVED** [review]
Good job!

**Pull request closed without merging**